source/fundamentals/aggregation/aggregation-expression-operations.txt

```diff
@@ -157,5 +155,77 @@ the Query API:
                                       then: "extreme temperature",
                                       else: "no advisory" } } } }
 
+String Operations
+~~~~~~~~~~~~~~~~~
+
+You can use string operations to examine and manipulate strings. You can
+perform a string operation on a value of type ``MqlString`` using the
+methods described in this section.
+
+The following table describes string operation methods
+available in the driver and corresponding expression operators in the
+Query API. The method names link to API documentation and the
+aggregation pipeline operator names link to Server manual documentation,
+which has definitions and examples.
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Java Method
+     - Aggregation Pipeline Operator
+
+   * - `append() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#append(com.mongodb.client.model.mql.MqlString)>`__
+     - :manual:`$concat </reference/operator/aggregation/concat/>`
+       
+   * - `toLower() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#toLower()>`__
+     - :manual:`$toLower </reference/operator/aggregation/toLower/>`
+
+   * - `toUpper() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#toUpper()>`__
+     - :manual:`$toUpper </reference/operator/aggregation/toUpper/>`
+
+   * - `length() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#length()>`__
+     - :manual:`$strLenCP </reference/operator/aggregation/strLenCP/>`
+
+   * - `lengthBytes() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#lengthBytes()>`__
+     - :manual:`$strLenBytes </reference/operator/aggregation/strLenBytes/>`
+
+   * - `substr() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#substr(int,int)>`__
+     - :manual:`$substrCP </reference/operator/aggregation/substrCP/>`
+
+   * - `substrBytes() <{+api+}/apidocs/mongodb-driver-core/com/mongodb/client/model/mql/MqlString.html#substrBytes(int,int)>`__
+     - :manual:`$substrBytes </reference/operator/aggregation/substrBytes/>`
+
+.. include:: /includes/fundamentals/agg-mql-diff.rst
+
+The example creates a lowercase username from an employee's last
+name and employee ID.
+
+The example concatenates the ``lastName`` and ``employeeID`` fields by
+using the ``append()`` method and makes this string lowercase by using
+the ``toLower()`` method. The resulting string is projected as the
+``username`` field. The following code shows a pipeline stage for
+this aggregation:
```

Comment:
Suggestion:

I think it would be good to present the example as a hypothetical rather than refer to it as "the example" without any context.


```suggestion
For example, suppose you need to generate lowercase usernames
for employees of a company by concatenating their last name and
ID.

To concatenate the ``lastName`` and ``employeeID`` fields, use the
append() string operation method. To convert them to lowercase, use
the toLower() string operation method. Use the project() method to
surface the result in the username field.
 
The following code shows the pipeline stage for this aggregation:
```